### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/APIC-586

### Changes included:

Update README format and content

## 🧪 Test

[Preview](https://github.com/algolia/api-clients-automation/blob/b4a6efbf6041088b3c3af86a1b8bea035f852108/README.md)

### CODE REVIEW

**Review by millotp — COMMENTED** [review]
Looks very pro !

File: README.md [hunk 1]
```diff
@@ -1,50 +1,78 @@
-# api-clients-automation
+<div align="center">
 
-**Make sure to have Docker installed so you don't have to install the tooling for every API clients.**
+# API Clients Automation
 
-## Setup repository tooling
+The Algolia API clients are generated from [OpenAPI specs](https://swagger.io/specification/), leveraging the open-source [openapi-generator](https://openapi-generator.tech/) tool.
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/09af048d-a19a-44db-9312-84172b73599e/deploy-status?branch=main)](https://app.netlify.com/sites/api-clients-automation/deploys) [![License](https://img.shields.io/badge/license-MIT-green.svg?style=flat-square)](./LICENSE)
```
[the remark this answers is not in the document]
Comment (shortcuts) [reply README.md:7]: yup just saw that :( it's a shame that cancelled is not properly reported

File: README.md [hunk 2]
```diff
@@ -1,50 +1,78 @@
...
 yarn docker:setup
 ```
 
 [Read more on our documentation](https://api-clients-automation.netlify.app/docs/contributing/setup-repository)
 
-## Contributing
+### CLI
 
-You can make changes locally and run commands through the docker container.
+The CLI allows your to make changes locally and run commands through the docker container.
```
Comment (millotp) [inline README.md:48]:
```suggestion
The CLI allows you to make changes locally and run commands through the docker container.
```

**Review by millotp — COMMENTED** [review]
last fix !

File: README.md (hunk 2 above)
Comment (shortcuts) [reply README.md:48]: 👁️ 👄 👁️ 🦅

**Review by millotp — APPROVED** [review]
nice :)

**Pull request merged**